gsoc2018-bharat/README.md

```diff
@@ -2,16 +2,27 @@
 
 This project aims to generate complex word embeddings for Out-Of-Vocabulary entities. After completion, the package will be able to generate pre-trained embeddings, improve them by generating embeddings on-the-fly, evaluate the benchmarks using pre-trained embeddings, and make the same evaluations on the imporoved embeddings.
 
-## Requirements
+### Getting started
 
-All the libraries used under this project are included in the file, requirements.txt. To install, just run the command
```

Comment:
I have the feeling that these changes belong to a previous version of the repo, so maybe we have to undo them